resources/puppetlabs/lein-ezbake/template/global/controller.sh.erb

```diff
@@ -105,10 +105,27 @@ params=("--user" "<%= EZBake::Config[:user] -%>" "--group" "<%= EZBake::Config[:
 if [ -n "$os_version" ]; then params+=("--os-version" "$os_version"); fi
 if [ -n "$os_dist" ]; then params+=("--dist" "$os_dist"); fi
 
+params+=('--description' 'Puppet Labs <%= EZBake::Config[:project] %>. Contains: <%= Pkg::Config.config[:description] -%>')
+
 <% unless EZBake::Config[:terminus_info].empty? -%>
 DESTDIR="$basepath/termini" bash install.sh termini
 params+=('--build-termini')
 params+=('--termini-chdir' "$basepath/termini")
+termini_description='Termini for <%= EZBake::Config[:project] %>. Contains terminus for:'
+
+<% comma_counter = 0 -%>
+<% EZBake::Config[:terminus_info].each do |proj_name, info| -%>
+<%   if comma_counter == 0
+       comma = ''
+     else
+       comma = ', '
+     end
+     comma_counter += 1
+-%>
+termini_description+='<%= comma -%><%= proj_name -%> (version <%= info[:version] -%>)'
+<%   end # if comma_counter == 0 -%>
```

Comment:
probably want this comment gone
also, this `end` goes with the `each`, right? so I think should be left aligned.